docs/content/developers/design-keuzes.md

```diff
@@ -190,3 +190,17 @@ RGBZ. BESLUIT krijgt een optionele relatie naar ZAAK.
 
 De verwachting is dat er later vergelijkbare designkeuzes gemaakt worden voor
 andere resources, zoals bijvoorbeeld klantcontacten.
+
+## Zaak afsluiten
+
+In de huidige ZDS 1.x standaard is er nog [geen eenduidig besluit genomen over hoe een zaak wordt afgesloten](https://discussie.kinggemeenten.nl/discussie/gemma/koppelvlak-zs-dms/afsluiten-van-een-zaak). 
+
+Wij hebben gekozen voor grofweg de volgende aanpak:
+* Objecttype STATUSTYPE krijgt een nieuw attribuut: `isEindStatus` (ja/nee). Er mag maximaal 1 eindstatus zijn per ZAAKTYPE.
+* Het attribuut `Einddatum` op ZAAK wordt alleen-lezen.
+* Bij het toekennen van een STATUSTYPE met `isEindStatus=ja` aan ZAAK wordt de `ZAAK.Einddatum` automatisch ingevuld met de waarde uit `STATUS.datumStatusGezet`.
+
+Het RGBZ wordt hier op aangepast.
+
+**TODO:**
+_Of de ZAAK hierna niet meer kan worden aangepast is wellicht praktisch niet handig maar zou wel het zuiverste zijn. Het is alleen technisch niet 100% af te dekken. Je kan namelijk een document toevoegen aan een zaak, zonder dat het ZRC "dit weet" aangezien de relatie (op dit moment) ligt bij het DRC._
```

Comment:
Joeri - misschien beter een voorbeeld gebruiken als BRP o.i.d. in plaats van een component waar we wel controle over hebben?

Reply:
Ja eens. Laten we het volgende maandag even face to face bespreken met iedereen, zodat het probleem, impact en oplossingsrichting voor iedereen helder is en we een klap er op geven. Dan passen we daarna deze PR aan.

Reply:
Ik zou bijna zeggen dat het probleem meer zit op de status afgesloten. Een zaak die afgehandeld is (er is één of meer besluiten genomen) kan daarna nog aangevuld worden met documenten, nagekomen stukken etc. Dat is de praktijk, daar doen we weinig aan. Alleen moet de zaak dan niet de status afgesloten krijgen, want dat is hij dan nog niet. 

De zaak is dan voor de client (zaakafhandelcomponent/ZAC) afgehandeld en staat niet meer op de takenlijst van de medewerker. In de ZRC is de zaak weliswaar afgehandeld maar nog niet afgesloten. Afsluiten is dan een administratieve stap die gedaan kan worden nadat alle gegevens netjes opgeslagen zijn, het dossier aangevuld is etc.  Per zaaktype zou bijvoorbeeld ingericht kunnen worden welke gegevens/dossierstukken minimaal aanwezig moeten zijn om een zaak af te sluiten. Toevoegen van dossierstukken kan ook door de postkamer gebeuren.

Misschien moeten we zelfs zeggen dat er na het afhandelen van een zaak een stap afronden/completeren zaakdossier komt. Wanneer het zaakdossier afgerond en compleet is kan de zaak afgesloten worden. Na afsluiten kan de zaak niet meer gewijzigd worden.

Reply:
Zoals hierboven genoemd is het niet wijzigen van een Zaak na afsluiten praktisch niet handig.

Het kan eventueel gelimiteerd worden, bijv.: na afsluiten mag je nog wel de status wijzigen naar een "niet-eindstatus", waarna je andere velden mag bewerken. Ik wil de API hier voor nu niet mee belasten, maar dit kan in een later stadium zeker nog er bij zonder dat alles omgegooid hoeft te worden.

Ik wil vooral de flow als design keuze hebben. Wat de praktische gevolgen wel/niet zijn moeten we even door User Stories laten dicteren.